doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Usage & Billing"
+description: "Provides an overview of various factors that can affect cost."
+menu:
+  main:
+    parent: "administration"
+    weight: 10
+---
+
+From the [console](https://console.materialize.com/) (`Admin` > `Usage &
+Billing`), administrators can access their invoice. The invoice provides
+Compute and Storage usage and cost information.
+
+## Cost Factors
+
+Various factors can affect your costs, including (but not limited to):
+
+- Each [replica for a cluster](/sql/create-cluster/#credit-usage). To see the
+  number of replicas for your clusters, click on `Clusters` in the console,
+
+- Creating a [materialized view](/concepts/views/#materialized-views) as
+  materialized view creation executes the underlying query and stores the
+  results to durable storage, and materialized views incrementally updates
+  results as inputs change. To see the views in a cluster, in the console's
+  `Clusters` view, select the cluster to inspect, and click on the `Materialized
+  Views` tab.
+
+  {{< note >}}
+
+  For more information on how incremental computations can lower the cost of
+  freshness, see [How Materialized can lower the cost of freshness for data
+  team](https://materialize.com/promotions/cost-of-freshness/?utm_campaign=General&utm_source=documentation).
```

Comment:
```suggestion
  teams](https://materialize.com/promotions/cost-of-freshness/?utm_campaign=General&utm_source=documentation).
```

Reply:
The "cost of freshness" is figurative, not sure I'd include this in a documentation page about billing.

Reply:
I think Jessica's suggestion of leading with the value may obviate the need for the notes. And can add the link to the paper as a see also or additional references section.